modules/tensor_mechanics/test/tests/crystal_plasticity/stress_update_material_based/exception_backstress.i

```diff
@@ -0,0 +1,227 @@
+[GlobalParams]
+  displacements = 'ux uy uz'
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  elem_type = HEX8
+[]
+
+[AuxVariables]
+  [./pk2]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./fp_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./rotout]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./e_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./gss]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./slip_increment]
+   order = CONSTANT
+   family = MONOMIAL
+  [../]
+  [./backstress]
+   order = CONSTANT
+   family = MONOMIAL
+  [../]
+[]
+
+[Modules/TensorMechanics/Master/all]
+  strain = FINITE
+  add_variables = true
+  generate_output = stress_zz
+[]
+
+[AuxKernels]
+  [./fp_zz]
+    type = RankTwoAux
+    variable = fp_zz
+    rank_two_tensor = plastic_deformation_gradient
+    index_j = 2
+    index_i = 2
+    execute_on = timestep_end
+  [../]
+  [./pk2]
+   type = RankTwoAux
+   variable = pk2
+   rank_two_tensor = second_piola_kirchhoff_stress
+   index_j = 2
+   index_i = 2
+   execute_on = timestep_end
+  [../]
+  [./e_zz]
+    type = RankTwoAux
+    variable = e_zz
+    rank_two_tensor = total_lagrangian_strain
+    index_j = 2
+    index_i = 2
+    execute_on = timestep_end
+  [../]
+  [./gss]
+    type = MaterialStdVectorAux
+    variable = gss
+    property = slip_resistance
+    index = 0
+    execute_on = timestep_end
+  [../]
+  [./slip_inc]
+   type = MaterialStdVectorAux
+   variable = slip_increment
+   property = slip_increment
+   index = 0
+   execute_on = timestep_end
+  [../]
+  [./backstress]
+   type = MaterialStdVectorAux
+   variable = backstress
+   property = backstress
+   index = 0
+   execute_on = timestep_end
+  [../]
+[]
+
+[BCs]
+  [./symmy]
+    type = DirichletBC
+    variable = uy
+    boundary = bottom
+    value = 0
+  [../]
+  [./symmx]
+    type = DirichletBC
+    variable = ux
+    boundary = left
+    value = 0
+  [../]
+  [./symmz]
+    type = DirichletBC
+    variable = uz
+    boundary = back
+    value = 0
+  [../]
+  [./tdisp]
+    type = FunctionDirichletBC
+    variable = uz
+    boundary = front
+    function = '0.1*t'
+  [../]
+[]
+
+[Materials]
+  [./elasticity_tensor]
+    type = ComputeElasticityTensorCP
+    C_ijkl = '1.684e5 1.214e5 1.214e5 1.684e5 1.214e5 1.684e5 0.754e5 0.754e5 0.754e5'
+    fill_method = symmetric9
+  [../]
+  [./stress]
+    type = ComputeMultipleCrystalPlasticityStress
+    crystal_plasticity_models = 'trial_xtalpl'
+    tan_mod_type = exact
+    rtol = 1e-6 # Constitutive stress residual relative tolerance
+    maxiter_state_variable = 50 # Maximum number of iterations for stress update
+
+    maximum_substep_iteration = 25 # Maximum number of substep iteration
+
```

Comment:
the MOOSE regression tests are used to check for correct code behavior more than for correct material modeling, and it is a hard requirement that the regression tests must run in under 2 seconds. With the tighter stress residual tolerance and large maximum iteration numbers, it's likely that this test will take longer than 2 seconds to run

Reply:
Okay, I'll comment this out and take the default

Reply:
resolved